src/conduits/java/com/enderio/conduits/common/conduit/block/ConduitBlock.java

```diff
@@ -263,7 +263,7 @@ private Optional<InteractionResult> handleYeta(ConduitBlockEntity conduit, Playe
             } else {
                 ConnectionState connectionState = conduit.getBundle().getConnectionState(hit.getDirection(), type);
                 if (connectionState == StaticConnectionStates.DISABLED) {
```

Comment:
To get this to work I also needed to change this to `if (connectionState.isConnection()) {`

If you'd agree with the change, please go ahead and make it and I'll merge the PR. Cheers!